spirv_msl.hpp

```diff
@@ -266,6 +270,7 @@ class CompilerMSL : public CompilerGLSL
 		uint32_t dynamic_offsets_buffer_index = 23;
 		uint32_t shader_input_wg_index = 0;
 		uint32_t device_index = 0;
+		uint32_t argument_buffer_offset = 0; // Allow the caller to specify the Metal translation should use argument buffers
```

Comment:
What does this do? "should use argument buffers" is another option.

Reply:
`argument_buffer_offset` is added to the buffer binding slot in `CompilerMSL::entry_point_args_argument_buffer`, to provide a way to adjust the binding slot of argument buffers. The commentary is misleading here, which might be the result of my rebasing on top of a more recent revision. Let me update this.